build/Repositories.props

```diff
@@ -7,6 +7,7 @@
 
   <ItemGroup>
     <Repository Include="Antiforgery" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="AspNetCoreModule" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
```

Comment:
We'll also need to update https://github.com/aspnet/AspNetCoreModule/blob/dev/test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj#L15 to point to the right build of packages if we intend for it to be built as part of our build process.